### PR TITLE
Bootswatch inheritance

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: mod/bootstrap/set/type/bootswatch_skin.rb [hunk 1]
```diff
@@ -1,15 +1,174 @@
...
+#
+# For the original bootswatch themes all those parts are hard-coded and the content
+# is taken from files.
+# The bootswatch theme content is taken directly from the files in the bootswatch
+# submodule. For the rest we use code file cards.
+# Cards of type "customized bootswatch skin" have the same structure but make the
+# variables and stylesheets part editable.
+#
+# Bootswatch theme cards are "asset inputters," meaning they generate a
+# Machine cards usually store all involved input cards of all nested levels in
```
Comment (xithan) [inline mod/bootstrap/set/type/bootswatch_skin.rb:27]: old wording - machines are gone

File: mod/assets/lib/card/assets.rb [hunk 2]
```diff
@@ -30,6 +30,15 @@ def refresh_assets force: false
         generate_asset_output_files if force
       end
 
+      def wipe
+        Auth.as_bot do
+          %i[asset_input asset_output].each do |field|
+            Card.search(right: field).each(&:delete!)
+            Virtual.where(right_id: field.card_id).destroy_all
```
Comment (xithan) [inline mod/assets/lib/card/assets.rb:37]: Did this do the trick?

Comment (ethn) [reply mod/assets/lib/card/assets.rb:37]: It works, but this isn't what solved the problem. The issue was in the variables card and happened even when re-eating from a fresh deck.  Still, nice to have around.

File: mod/bootstrap/set/type/bootswatch_skin.rb (hunk 1 above)
Comment (ethn) [reply mod/bootstrap/set/type/bootswatch_skin.rb:27]: Right. I started in on this edit but never finished it. Thanks.

**Pull request merged**